src/plonkish/backend/halo2.rs

```diff
@@ -187,193 +187,207 @@ impl<F: PrimeField + From<u64> + Hash> ChiquitoHalo2<F> {
             plonkish_ir: circuit,
             advice_columns: Default::default(),
             fixed_columns: Default::default(),
-            instance_column: Default::default(),
             ir_id,
         }
     }
 
-    fn configure_columns_sub_circuit(&mut self, meta: &mut ConstraintSystem<F>) {
-        let mut advice_columns = HashMap::<UUID, Column<Advice>>::new();
-        let mut fixed_columns = HashMap::<UUID, Column<Fixed>>::new();
+    fn configure_halo2_columns(&mut self, meta: &mut ConstraintSystemBuilder<F>) {
```

Comment:
@alxkzmn why is called meta here but cs_builder somewhere. meta was the name we were using in the times of zkevm

Reply:
I'll change it to constraint system too, meta is not a very meaningful name but it is used in Halo2 code.